src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle(String color, int radius) {
+        this.radius = radius;
+        this.setColor(Color.valueOf(color));
```

Comment:
you should initialize color field using constructor of Figure, fix it in every figure

Reply:
fixed